dipy/viz/regtools.py

```diff
@@ -365,10 +365,10 @@ def overlay_slices(L, R, slice_index=None, slice_type=1, ltitle='Left',
         the type of slice to be extracted:
         0=sagital, 1=coronal (default), 2=axial.
     ltitle : string (optional)
-        the string to be written as title of the left image. By default,
+        the string to be written as the title of the left image. By default,
```

Comment:
Not sure, for this one...

Reply:
Since 'title' was used in multiple places, I thought it might need emphasis and as a result, I added "the" before the title.
Shall I revert it?